src/Temporalio/Client/HttpConnectProxyOptions.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+
+namespace Temporalio.Client
+{
+    /// <summary>
+    /// HTTP connect proxy options for Temporal connections.
+    /// </summary>
+    public class HttpConnectProxyOptions : ICloneable
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HttpConnectProxyOptions"/> class.
+        /// </summary>
+        /// <param name="targetHost">A 'host:port' string representing the target to proxy through.</param>
+        public HttpConnectProxyOptions(string targetHost) => TargetHost = targetHost;
```

Comment:
There needs to be a default no-arg constructor too

Reply:
Updated, thanks for the education on this!